enterprise-suite/es-grafana/akka-exporter.json

```diff
@@ -1,13 +1,13 @@
 [  
   {"graphName":"Mailbox Size", "promQL":["max by (actor)(akka_actor_mailbox_size{workload=\"$workloadId\"})"]},
   {"graphName":"Mailbox Time", "promQL":["max by (actor)(akka_actor_mailbox_time_ns{workload=\"$workloadId\",quantile=\"0.99\"})"]},
-  {"graphName":"Actor Failures", "promQL":["sum by (actor) (akka_actor_actor_failure{workload=\"$workloadId\"})"]},
-  {"graphName":"Processed Messages", "promQL":["sum by (actor)(irate(akka_actor_processed_messages{workload=\"$workloadId\"}[1m]))"]},      
+  {"graphName":"Actor Failures per 15 Minutes", "promQL":["sum by (actor) (rate(akka_actor_actor_failure{workload=\"$workloadId\"}[15m]))"]},
```

Comment:
"Actor Failure Rate"  to be consistent, rate is per/second on the data over the past 15 min.  You'll get a better looking graph with irate, which won't look so smoothed and will pop up sharply when there's an error.

I'd change this to "Actor Failure Rate" and irate( ... [1m])